### ISSUE TITLE
Incorrect Color

### ISSUE THREAD

**Issue by derek-skinner** [jariz/vibrant.js#8]
When I run a test using Vibrant.js on just a single solid color jpeg, the output color is incorrect.

Test was performed using this Fiddle: http://jsfiddle.net/oskar/3479gj9e/light/
### Color used for test:

Hex #313788
### Output Colors:

Circle 1:
Hex #343c8c

Circle 3:
Hex #383c8c

![vibrantjs_screenshot](https://cloud.githubusercontent.com/assets/801299/7970343/40ff2670-09f2-11e5-84af-4aae79b3b04d.jpg)

**Comment by jariz** [comment]
This is because Vibrant (just like it's Palette android equivalent) will try to generate colors (to be specific, either `VibrantSwatch` or `DarkVibrantSwatch`) when they don't exist.
Vibrant was mostly made for photographs aka things with a lot of different colors in them. You can't really blame it for not picking up a single color. I suggest looking into [ColorThief](https://github.com/lokesh/color-thief) (on which vibrant was based) to get a palette / the most used color.

TL;DR: It's a feature, not a bug.
